lib/std/rand.zig

```diff
@@ -749,29 +749,31 @@ pub const Gimli = struct {
     random: Random,
     state: std.crypto.core.Gimli,
 
-    pub fn init(init_s: u64) Gimli {
+    pub const secret_seed_length = 32;
```

Comment:
```suggestion
    pub const secret_seed_length = std.crypto.core.Gimli.BLOCKBYTES - std.crypto.core.Gimli.RATE;
```
would make it more clear that the seed is exactly as long as the preserved state between calls.

Reply:
The fact that it matches the capacity is more of a coincidence than a requirement. We could have chosen a 128-bit seed, filled the entire state, or even absorbed an arbitrary-large key. `secret_seed_length` is really a parameter of the RNG, not a requirement of the underlying permutation.